<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Tryb Auto Multi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>280</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Tryb Manualny</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>410</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyjście</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Tryb Auto Solo</string>
    </property>
   </widget>
   <widget class="QWidget" name="Auto2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>861</width>
      <height>611</height>
     </rect>
    </property>
    <widget class="QPushButton" name="auto2_back">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>450</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Wstecz</string>
     </property>
    </widget>
    <widget class="QPushButton" name="auto2_start">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>360</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="auto2_select1">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>50</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Przeglądaj</string>
     </property>
    </widget>
    <widget class="QPushButton" name="auto2_select2">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>100</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Przeglądaj</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="auto2_sel_text1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>49</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="auto2_sel_text2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>99</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ścieżka Odczytu:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ścieżka Zapisu:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Threads">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>190</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>190</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ilość wątków:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>200</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Metoda Detekcji:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="auto2_met1">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>190</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Obejmowanie ramek</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="auto2_met2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>210</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Metoda Gaussa</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>0</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tryb Manualny</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="FrameSkip">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>280</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="GapSize">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>310</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="MotionLen">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>340</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="BeforeMove">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>370</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="AfterMove">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>280</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Area">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>310</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="History">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>340</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Mixtures">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>370</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>280</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pominięcie ramek:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>310</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Długość przerwy:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>370</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wyprzedzanie ruchu:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>280</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Przedłużanie ruchu:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>310</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pole obiektu:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>340</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Długość ruchu:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>340</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Historia:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>370</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ilość mieszanin:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>280</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>(0-N)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>310</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>(0-N)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>340</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>(1-N)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>370</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>(0-N)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>280</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>(0-N)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>310</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>(0-N)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>340</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>(10-N)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>370</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>(2-7)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="auto2_def">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>460</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Domyślne</string>
     </property>
    </widget>
    <widget class="QWidget" name="zapis" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>261</width>
       <height>80</height>
      </rect>
     </property>
     <widget class="QRadioButton" name="zapis1">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Pamięć RAM</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="zapis2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Dysk</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Przechowywanie danych:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Man" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>851</width>
      <height>601</height>
     </rect>
    </property>
    <widget class="QPushButton" name="man_back">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>500</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Wstecz</string>
     </property>
    </widget>
    <widget class="QPushButton" name="man_next">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>420</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Następny</string>
     </property>
    </widget>
    <widget class="QPushButton" name="man_save">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>340</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Zapisz</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>540</y>
       <width>381</width>
       <height>19</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="man_play">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>500</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>play</string>
     </property>
    </widget>
    <widget class="QPushButton" name="man_5plus">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>500</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>+5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="man_5minus">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>500</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="man_1plus">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>500</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>+1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="man_1minus">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>500</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>501</width>
       <height>401</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="go_start">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>170</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>idź na
 początek fragmentu</string>
     </property>
    </widget>
    <widget class="QPushButton" name="go_end">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>220</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>idź na
 koniec fragmentu</string>
     </property>
    </widget>
    <widget class="QPushButton" name="set_start">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>170</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>ustaw
 początek fragmentu</string>
     </property>
    </widget>
    <widget class="QPushButton" name="set_end">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>220</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>ustaw
 początek fragmentu</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Auto1" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>861</width>
      <height>601</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tryb Automatyczny</string>
     </property>
    </widget>
    <widget class="QPushButton" name="auto1_back">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>500</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Wstecz</string>
     </property>
    </widget>
    <widget class="QPushButton" name="auto1_add">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>180</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Dodaj</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>501</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>100</y>
       <width>20</width>
       <height>441</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Lp.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tytuł</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>110</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="Lp">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>61</width>
       <height>441</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="Name">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>341</width>
       <height>441</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="Status">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>100</y>
       <width>111</width>
       <height>441</height>
      </rect>
     </property>
    </widget>
    <zorder>Lp</zorder>
    <zorder>Status</zorder>
    <zorder>Name</zorder>
    <zorder>label_10</zorder>
    <zorder>auto1_back</zorder>
    <zorder>auto1_add</zorder>
    <zorder>line</zorder>
    <zorder>line_3</zorder>
    <zorder>label_28</zorder>
    <zorder>label_29</zorder>
    <zorder>label_30</zorder>
   </widget>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>Auto1</zorder>
   <zorder>Auto2</zorder>
   <zorder>Man</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>860</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdycja">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="actionO_programie"/>
    <addaction name="actionAutorzy"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuEdycja"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionBrak">
   <property name="text">
    <string>O programie</string>
   </property>
  </action>
  <action name="actionO_programie">
   <property name="text">
    <string>O programie</string>
   </property>
  </action>
  <action name="actionAutorzy">
   <property name="text">
    <string>Autorzy</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
